src/Compilers/CSharp/Test/IOperation/IOperation/IOperationTests_IConditionalAccessExpression.cs

```diff
@@ -1236,5 +1236,127 @@ void M1(C input1, C input2, C input3)
 
             VerifyFlowGraphAndDiagnosticsForTest<BlockSyntax>(source, expectedGraph, expectedDiagnostics);
         }
+
+        [Fact]
+        public void NestedConditionalAccess()
+        {
+            var comp = CreateCompilation(@"
+class C
+{
+    C Prop { get; set; }
+    object M2(C c) => throw null;
+    void M(C c)
+    /*<bind>*/{
+        _ = c?.Prop.M2(c?.Prop);
+    }/*</bind>*/
+}");
+
+            VerifyFlowGraphAndDiagnosticsForTest<BlockSyntax>(comp, @"
+Block[B0] - Entry
+    Statements (0)
+    Next (Regular) Block[B1]
+        Entering: {R1} {R2}
+.locals {R1}
+{
+    CaptureIds: [1]
+    .locals {R2}
+    {
+        CaptureIds: [0] [2] [4]
+        Block[B1] - Block
```

Comment:
>Block[B1] - Block [](start = 8, length = 17)

I think we actually should have a nested region here. it should enclose B1, B2 and own capture [0].